mu/modes/circuitpython.py

```diff
@@ -72,6 +72,8 @@ class CircuitPythonMode(MicroPythonMode):
         (0x239A, 0x80AC, None, "Unexpected Maker FeatherS2"),
         (0x303A, 0x8002, None, "Unexpected Maker TinyS2"),
         (0x054C, 0x0BC2, None, "Spresense"),
+        (0x303A, 0x80AA, None, "Franzininho WiFi Wroom"),
+        (0x303A, 0x80AD, None, "Franzininho WiFi Wrover"),        
```

Comment:
I think the reason the CI is complaining is that there are extra white spaces at the end of this line.
You can run the tests locally to ensure everything passes with: `make check` (or `python make.py check`)

Reply:
@carlosperate I removed the extra white space.

Thanks for the feedback